Aurora.py

```diff
@@ -26,37 +31,37 @@
 if options.portname is not None:
     port = options.portname
 else:
-    print "Usage: python Aurora.py -p <port name>"
-    print "(ex.: python Aurora.py -p /dev/ttyACM0)"
+    print("Usage: python Aurora.py -p <port name>")
+    print("(ex.: python Aurora.py -p /dev/ttyACM0)")

```

Comment:
doesn't that need the from future import print_function to work on py27?

Reply:
'from future import print_function' hasn't been needed in a while. I can recall since when but it certainly has been for a while.

Reply:
you're right, a current py27 does not need it.

maybe it was a thing with < 2.7 or early 2.7.